index.html

```diff
@@ -250,90 +268,96 @@ <h3 class="about-heading">About me</h3>
-            <img src="./images/Micro/skills.png" alt="bullet" /> Skills
-          </h3>
-          <ul class="semi-list">
-            <li>Codekit</li>
-            <li>Github</li>
-            <li>Codepen</li>
-            <li>Gitlab</li>
-            <li>Terminal</li>
-          </ul>
+          <div class="sub-skill-container">
+            <h3 class="head-skill-list">
+              <img src="./images/Micro/language.png" alt="bullet" /> Language
+            </h3>
+            <ul class="semi-list">
+              <li>Javascript</li>
+              <li>Ruby</li>
+              <li>Html</li>
+              <li>Css</li>
+            </ul>
+          </div>
+          <div class="sub-skill-container">
+            <h3 class="head-skill-list">
+              <img src="./images/Micro/frameworks.png" alt="bullet" />
+              Frameworks
+            </h3>
+            <ul class="semi-list">
+              <li>Bootstrap</li>
+              <li>Ruby on Rails</li>
+              <li>RSPec</li>
+              <li>CapyBara</li>
+              <li>Selenium</li>
+            </ul>
+          </div>
+          <div class="sub-skill-container">
+            <h3 class="head-skill-list">
+              <img src="./images/Micro/skills.png" alt="bullet" /> Skills
+            </h3>
+            <ul class="semi-list">
+              <li>Codekit</li>
+              <li>Github</li>
+              <li>Codepen</li>
+              <li>Gitlab</li>
+              <li>Terminal</li>
+            </ul>
+          </div>
         </article>
       </div>
     </section>
     <footer class="contact-section">
-      <p class="contact-message">
-        I'm always interested in hearing about new projects, so if you'd like to
-        chat please get in touch.
-      </p>
-      <form
-        action="https://formspree.io/f/mbjblgga"
-        method="post"
-        class="contact-form"
-      >
-        <ul class="contact-form-section">
-          <li>
-            <label for="name" class="name-section"></label
-            ><input
-              type="text"
-              name="user_name"
-              id="name"
-              placeholder="Full name"
-              maxlength="30"
-              required
-            />
-          </li>
+      <div class="contact-message-and-form">
+        <p class="contact-message">
+          I'm always<br />interested in hearing<br />about new projects,<br />so
+          if you'd like to<br />chat please get in<br />touch.
+        </p>
+        <form
+          action="https://formspree.io/f/mbjblgga"
+          method="post"
+          class="contact-form"
+        >
+          <ul class="contact-form-section">
+            <li>
+              <label for="name" class="name-section"></label
+              ><input
+                type="text"
+                name="user_name"
+                id="name"
+                placeholder="Full name"
+                maxlength="30"
+                required
+              />
```

Comment:
### Your Form
![your-design](https://user-images.githubusercontent.com/93039359/187049670-2524b9a9-f233-4180-8db8-e51a3dfcc0f6.png)

### Figma Form
![Screenshot from 2022-08-28 03-06-39](https://user-images.githubusercontent.com/93039359/187049672-d9c8759f-2434-4b0c-a6a4-62e8541242f4.png)

- [ ] You can see that the Figma form is expanding on the whole screen in mobile version but your fields are not as the previous reviewer suggested. Please make it look like the figma template :blush: to make your project look professional :+1:

**Hint: Remove the widths of input fields in the mobile version then your input fields will expand and look like the Figma template :blush:**

Reply:
i have just done that

Reply:
thanks for pointing it out